src/schema/schema.graphql

```diff
@@ -2063,6 +2063,10 @@ input DismissInput {
 	dismissedAt: Float!
 }
 
+input AddMsgInput {
+	folderId: ID!
+	attachmentId: String!
```

Comment:
Would it be good if we use `ID` for `attachmentId` instead of `String`. I saw another place where `String` is being used, though `ID` is also kind of `String` here.

Reply:
yes it should be ID

Reply:
Fixed